examples/pxScene2d/src/pxResource.cpp

```diff
@@ -24,6 +24,7 @@ pxResource::~pxResource()
     // if there is a previous request pending then set the callback to NULL
     // the previous request will not be processed and the memory will be freed when the download is complete
     mDownloadRequest->setCallbackFunctionThreadSafe(NULL);
+    delete mDownloadRequest;
```

Comment:
Multiple objects with multiple threads have pointers to this pxFileDownloadRequest object (stored in mDownloadRequest).  It should be deleted in either pxFileDownloader::downloadFile()  or pxResource::onDownloadComplete() only to avoid double frees.